games/0.yaml

```
@@ -218,6 +218,23 @@
   - MIT
   updated: '2023-09-28'
 
+- name: 2048 (Ebitengine Demo)
```

```suggestion
- name: Ebitengine
```
this repo contains many examples so should be named after the repo